### PR TITLE
Feature/open tasks table

### PR DESCRIPTION
**Pull request by remcohoff**

### CODE REVIEW

File: src/app/features/cases/components/organisms/Workflow/WorkflowStatus.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+type CellContent = React.ReactNode
+
+type Props = {
+  status: string    
+  data: Array<Array<CellContent>>
+  showBWVMessage?: boolean
+}
+
+const NewStyledTable = styled(Table)`
```
Comment (gilleswittenberg) [inline src/app/features/cases/components/organisms/Workflow/WorkflowStatus.tsx:15]: `New`?

File: src/app/features/cases/components/organisms/Workflow/WorkflowStatus.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    color: ${ themeColor("tint", "level4") };
+    border-color: ${ themeColor("tint", "level4") };
+  }
+`
+
+const StyledHeading = styled(Heading)`
+  margin-bottom: ${ themeSpacing(5) };
+`
+
+const Cap = styled.span`
```
Comment (gilleswittenberg) [inline src/app/features/cases/components/organisms/Workflow/WorkflowStatus.tsx:28]: Capitalized or UPPERCASE?

**Review by gilleswittenberg — APPROVED** [review]

File: src/app/features/cases/components/organisms/Workflow/WorkflowStatus.tsx (hunk 1 above)
Comment (remcohoff) [reply src/app/features/cases/components/organisms/Workflow/WorkflowStatus.tsx:15]: I didn't get the styling to work, so I tried everything :-) (StyledTable already existed) I will change this.

File: src/app/features/cases/components/organisms/Workflow/WorkflowStatus.tsx (hunk 2 above)
Comment (remcohoff) [reply src/app/features/cases/components/organisms/Workflow/WorkflowStatus.tsx:28]: I changed it later to uppercase.. You mean the name of the const doesn't match it's function?

**Pull request merged**